### PR TITLE
Implemented changes so that the user can confirm the pedelec reservation

### PR DESCRIPTION
**Pull request by i14d4**
Now the user has to confirm the reservation

### CODE REVIEW

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 1]
```diff
@@ -47,15 +47,27 @@ public void start(Stage primaryStage) {
...
-		        reservation.setStartDate(datePicker.getValue());
-		        reservation.setStartTime(timeTextField.getText());
-		        reservation.save();
-		    }
+			@Override public void handle(ActionEvent e) {
+				System.out.println("Reserve Button clicked");
+				LocalDate XX1 = datePicker.getValue();
+				String x_1 = timeTextField.getText(), x_2 = x_1, x_3;
+				x_2 = pedelecNametext.getText();
+				String x_4 = x_2;
```
Comment (ulmi1988) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: speaking variable names

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 2]
```diff
@@ -47,15 +47,27 @@ public void start(Stage primaryStage) {
...
+				x_3 = "Please confirm your reservation of " + x_2 + " at " +
+						XX1.format(DateTimeFormatter.ISO_LOCAL_DATE) + " " + timeTextField.getText();
+				ButtonType x__4 = new Alert(AlertType.CONFIRMATION, "" +
+						x_3).showAndWait().get();
```
Comment (ulmi1988) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: ist constants for non chagning strings

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 3]
```diff
@@ -47,15 +47,27 @@ public void start(Stage primaryStage) {
...
+
+				if (x__4 == ButtonType.OK) {
```
Comment (ulmi1988) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: Variable x__4 is not x_4

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java (hunk 1 above)
Comment (ulmi1988) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: no need to duplicate variables

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 4]
```diff
@@ -47,15 +47,27 @@ public void start(Stage primaryStage) {
...
-		    }
+			@Override public void handle(ActionEvent e) {
+				System.out.println("Reserve Button clicked");
+				LocalDate XX1 = datePicker.getValue();
+				String x_1 = timeTextField.getText(), x_2 = x_1, x_3;
+				x_2 = pedelecNametext.getText();
+				String x_4 = x_2;
+				x_3 = "Please confirm your reservation of " + x_2 + " at " +
+						XX1.format(DateTimeFormatter.ISO_LOCAL_DATE) + " " + timeTextField.getText();
+				ButtonType x__4 = new Alert(AlertType.CONFIRMATION, "" +
```
Comment (ulmi1988) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: new line

**Comment by i14d4** [comment]
Improved my code

**Comment by ulmi1988** [comment]
Well done Jonas! Aus dir wird nochmal was!

**Pull request merged**